clio-dataaccess-model/src/main/scala/org/broadinstitute/clio/server/dataaccess/elasticsearch/DocumentWgsCram.scala

```diff
@@ -0,0 +1,61 @@
+package org.broadinstitute.clio.server.dataaccess.elasticsearch
+
+import java.time.OffsetDateTime
+
+import org.broadinstitute.clio.util.model.{DocumentStatus, Location, UpsertId}
+
+case class DocumentWgsCram(
+  upsertId: UpsertId,
+  entityId: String,
+  location: Location,
+  project: String,
+  sampleAlias: String,
+  version: Int,
+  documentStatus: Option[DocumentStatus] = None,
+  pipelineVersion: Option[Long] = None,
+  workflowStartDate: Option[OffsetDateTime] = None,
+  workflowEndDate: Option[OffsetDateTime] = None,
+  cramMd5: Option[String] = None,
+  cramSize: Option[Long] = None,
+  cramPath: Option[String] = None,
+  craiPath: Option[String] = None,
+  cramMd5Path: Option[String] = None,
+  logPath: Option[String] = None,
+  fingerprintPath: Option[String] = None,
+  cromwellIdPath: Option[String] = None,
```

Comment:
I wonder why we store the path to the cromwell ID here instead of the ID itself, which would be fewer characters anyway.